galaxy/wrapper/computeMatrix.xml

```diff
@@ -85,12 +79,14 @@
                 help="Label to use in the output."/>
         </repeat>
 
-        <param name="scoreFile" format="bigwig" type="data"
```

Comment:
I have no clue how multiple="True" is handled in Galaxy, it's not documented. We need the quotes to be handled correctly and my way guarantees that.